dashboard/modules/job/job_consts.py

```diff
@@ -0,0 +1,2 @@
+JOB_CHANNEL = "JOB"
+RETRY_GET_ALL_JOB_INFO_INTERVAL_SECONDS = 2
```

Comment:
I think this can be longer than 2 seconds as it doesn't have to be really live?

Reply:
The job info will be obtained from two ways:
1. `GetAllJobInfo` from GCS. _(RETRY_GET_ALL_JOB_INFO_INTERVAL_SECONDS)_
2. Subscribe job channel from Redis.

Job head will long running in 2 after 1 is finished. Set a larger value of `RETRY_GET_ALL_JOB_INFO_INTERVAL_SECONDS` is useless for many error handling, e.g. the result of `GetAllJobInfo` is exceeded the RPC data size limit.

We can adjust the value as needed.